### ISSUE TITLE
[FEATURE]: PageNumber Auto extension: Exclude tagged pages from count

### ISSUE THREAD

**Issue by DoctorDan1986** [naturalcrit/homebrewery#513]
I like to make the first page a cover page for my own stylistic purposes.  It also helps when exporting as a PDF.  However, the pageNumber Auto feature naturally counts this first page as page 1, which throws off the auto count for what I want.

I was thinking that you might include a method to exclude certain pages from the output, something like a 

`<div class='pageNumber null'></div>`
or 
`<div class='pageNumber skip'></div>`

The effect I'm looking for is that any page with this class will have a page counter value of 0, so that if one, say, put this on the first page of the brew, and used the pageNumber Auto on page 2, it would exclude page 1 from the count and start the count at 1 on page 2.

### Additional Details

**Share Link** :

or

**Brew code to reproduce** : <details><summary>Click to expand</summary><code><pre>

PASTE BREW CODE HERE

</pre></code></details>

**Comment by Gazook89** [comment]
I couldn't figure out how to make a snippet that you could place on a particular page to make the count be skipped or reset, but here is a CSS method for resetting the page counter.  You have to target the page you want by page number, so the downside is that any rearranging of your text means you will need to double check the reset page is still targeted....Not likely that big of deal in this case, because it is unlikely the first page or two are going to get jostled around much.

CSS:
```
.page#p1 {
	counter-reset:phb-page-numbers;	
}
```
This could be made into a snippet if approved by the powers that be.

**Comment by Gazook89** [comment]
@calculuschild --- I think this goes one of two ways:  a snippet with the above is added to the Style Editor menus (which I can add very quickly), or just close it.    

Besides for the css snippet above, I don't see how this would be possible without big changes, such as custom markdown tokens(?).  I imagine that it is vanishingly rare that users would want to skip a page in their count, except in the first few pages in which case `counter-reset` will work.

**Comment by 5e-Cleric** [comment]
wouldn't this be solved with a snippet and `.page:has(.counterReset)`?

I know this is a 6yr old issue

**Comment by G-Ambatte** [comment]
> wouldn't this be solved with a snippet and `.page:has(.counterReset)`?
> 
> I know this is a 6yr old issue

Something like this:

```
.page:has(.pageCounterSkip) {
    counter-increment: phb-page-numbers 0;
}
```

See example brew: https://homebrewery.naturalcrit.com/share/Q6xWK06zBUa_

A far more difficult question is then about how to handle the footer mirroring to match the new page numbering.

**Comment by 5e-Cleric** [comment]
yea, you can't do nth-of-class() which would be awesome

**Comment by calculuschild** [comment]
There may be something with the `~` sibling selector...? https://stackoverflow.com/questions/5545649/can-i-combine-nth-child-or-nth-of-type-with-an-arbitrary-selector

**Comment by 5e-Cleric** [comment]
You are absolutely right, Selectors Level 4 introduces `:nth-child(An+B [of S]? )`

> "The :nth-child(An+B [of S]? ) pseudo-class notation represents elements that are among An+Bth elements from the list composed of their [inclusive siblings](https://dom.spec.whatwg.org/#concept-tree-inclusive-sibling) that match the [selector list](https://www.w3.org/TR/selectors-4/#selector-list) S, which is a [<complex-selector-list>](https://www.w3.org/TR/selectors-4/#typedef-complex-selector-list) [parsed as a forgiving selector list](https://www.w3.org/TR/selectors-4/#parse-as-a-forgiving-selector-list). If S is omitted, it defaults to *|*."

This means that we will be able to do it eventually. unfortunately, i did not find much intel on support yet.
Iirc, support will start from Safari, not from Chrome, so we might have to wait.

Sources:
https://developer.chrome.com/articles/css-nth-child-of-s/
https://www.w3.org/TR/selectors-4/#nth-child-pseudo

**Comment by 5e-Cleric** [comment]
Happy to reopen this, because support for `:nth-child(An+B [of S]? )` Has reached chrome

**Comment by dbolack-ab** [comment]
Is the above still how we wish to handle this, or would it be more suitable to use something like  #3258?

**Comment by 5e-Cleric** [comment]
An obvious idea would be to have some UI option, but this is not a WYSIWYG editor, and therefore, i'd say a css or markdown approach is preferred, as we did with the ToC include/exclude method. Can't remember the details about this issue right now, but there was some issue i can't remember.

**Comment by 5e-Cleric** [comment]
Closed by #3639
